packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -489,7 +495,7 @@ const Sidebar: React.FC<RouteComponentProps & SidebarProps> = props => {
           Fluent <span style={gradientTextStyles}>UI</span>
         </Text>
         <VersionDropdown width={props.width} />
-        <CopyToClipboard value={`yarn add ${pkg.name}@${pkg.version}`} timeout={3000}>
+        <CopyToClipboard value={`yarn add ${pkg.name}@${version}`} timeout={3000}>
```

Comment:
```suggestion
        {process.env.NIGHTLYRELEASE ? null : <CopyToClipboard value={`yarn add ${pkg.name}@${version}`} timeout={3000}>}
```

I would rather hide it at all

Reply:
I used ` {process.env.NIGHTLYRELEASE === 'true' ? null : xxx` for similar reason as previous comment, can't set NIGHTLYRELEASE to undefined in azure pipeline :(

Reply:
empty string will be falsy right ?

Reply:
Right, good point. let me change this to be just a simple `process.env.NIGHTLYRELEASE ? `